### PR TITLE
[release] | [mxnet] Release logic for MX16

### PR DESCRIPTION
**Pull request by YYStreet**
*Issue #, if available:*

## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [release] | [tensorflow, mxnet]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release_images.yml [hunk 1]
```diff
@@ -4,22 +4,38 @@ release_images:
     framework: "tensorflow"
     version: "2.2.0"
     training:
-      device_types: ["cpu", "gpu"]
```
Comment (arjkesh) [inline release_images.yml:7]: I thought we were going to re-release inference cu102 as well?

Comment (YYStreet) [reply release_images.yml:7]: TF22 cu102 Inference images have already been released together with TF22 cu101 Training images yesterday.

Comment (YYStreet) [reply release_images.yml:7]: Added them back

File: release_images.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ release_images:
       device_types: ["cpu", "gpu"]
       python_versions: ["py37"]
       os_version: "ubuntu18.04"
-      cuda_version: "cu101"
+      cuda_version: "cu102"
```
Comment (arjkesh) [inline release_images.yml:10]: can force_release be moved?

Comment (YYStreet) [reply release_images.yml:10]: Removed. Thanks

**Comment by arjkesh** [comment]
Approve, can't figure out how to approve from the mobile app

**Pull request merged**